plugins/pixart-rf/fu-pxi-firmware.c

```diff
@@ -38,26 +40,33 @@ fu_pxi_rf_firmware_parse (FuFirmware *firmware,
 			     "firmware invalid, too small!");
 		return FALSE;
 	}
+	/* Get fw header */
+	if (!fu_memcpy_safe (fw_header, 32, 0x0,
+			     buf, bufsz, bufsz - 32,
+			     32, error)) {
```

Comment:
use sizeof(fw_header) rather than `32` pls.

Reply:
fixed(using macro)

Reply:
I think sizeof(fw_header) is actually still safer.